Connected new components with General GeneralTab

kamil-matula opened the pull request:
DONE:
- Modified PokemonGeneralInfo and PokemonStats components so now they use the UseContext method instead of arguments to display the data,
- Modified General screen so it displays both of these components in a ScrollView,
- Added possibility to return to Home Screen by clicking the arrow with "Pokedex" title in the left upper corner of the details screen,
- Modified Data.js file to return just numbers of weight and height without any letters + uncommented stats,
- Modified main screen's pokemons' tiles: the id is displayed in format eg. "#001" and the types have some space between them.

TO DO:
- General Screen needs a lot of styling - new components don't fit there well,
- Tabs in Details should have color of pokemon instead of default one,
- Clicking back button should transport into Home screen, not another tab (which happens if we click more than one tab in details screen).

![image](https://user-images.githubusercontent.com/43967269/116995213-1465dc80-acda-11eb-9aaa-57ca6780c50d.png)
![image](https://user-images.githubusercontent.com/43967269/116995352-4119f400-acda-11eb-9b15-2d35bf0054f6.png)

kamil-matula committed:
Connected new components with General GeneralTab

DONE:
- Modified PokemonGeneralInfo and PokemonStats components so now they use the UseContext method instead of arguments to display the data,
- Modified General screen so it displays both of these components in a ScrollView,
- Added possibility to return to Home Screen by clicking the arrow with "Pokedex" title in the left upper corner of the details screen,
- Modified Data.js file to return just numbers of weight and height without any letters + uncommented stats,
- Modified main screen's pokemons' tiles: the id is displayed in format eg. "#001" and the types have some space between them.

TO DO:
- General Screen needs a lot of styling - new components don't fit there well,
- Tabs in Details should have color of pokemon instead of default one,
- Clicking back button should transport into Home screen, not another tab (which happens if we click more than one tab in details screen).

Blazevarjo commented on src/components/PokemonCard.js:
```diff
@@ -60,7 +62,7 @@ const styles = StyleSheet.create({
     left: 10,
     top: 10,
-    fontSize: 18,
+    fontSize: 15,
```
it is 18 in figma

Blazevarjo commented on src/screens/details/General.js:
```diff
@@ -25,9 +28,11 @@ export default function General() {
     flex: 1,
     alignItems: 'center',
     justifyContent: 'center',
+    marginTop: 10,
+    marginBottom: 10,
```
There is style called marginVertical

kamil-matula committed:
Cleaned up the code

Changed marginBottom and marginTop into marginVertical, reverted changing fontSize of ID in PokemonCard.
